src/gr11review/part1/Review2.java

```diff
@@ -1 +1,36 @@
 package gr11review.part1;
+
+import java.util.Scanner;
+
+public class Review2{
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("0 - print a joke about your hair");
+        System.out.println("1 - print a joke about your feet");
+        System.out.println("2 - print a joke about your clothes");
+        System.out.println("3 - print a joke about your teacher");
+
+
+        System.out.println("choose a menu option: ");
```

Comment:
Fails invalid option.  `"choose a menu option: "` print should be `print` not `println`

See test results
https://github.com/SACHSTech/native-java-and-gr-11-review-assignment-amaan2106/runs/2623634154#step:3:46